lib/components/FileAttachment/Attachment.jsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState } from "react";
+
+import { MenuVertical, File } from "@bigbinary/neeto-icons";
+
+import Dropdown from "components/Common/Dropdown";
+import Input from "components/Common/Input";
+import MenuButton from "components/Common/MenuButton";
+
+import { DROP_DOWN_OPTIONS_KEYS, KEYS } from "./constant";
+import { validFileName } from "./utils";
+
+const { Menu, MenuItem } = Dropdown;
+
+const Attachment = ({ attachment, dropDownOptions = [] }) => {
+  const [error, setError] = useState("");
+  const [activeFile, setActiveFile] = useState({});
```

Comment:
Since the `Attachment.jsx` component correspond to a single attachment component, having a state named `activeFile` does not make any sense. Please re-think the logic. Let me know if you need help.